superset/common/query_context.py

```diff
@@ -86,10 +86,9 @@ def __init__(
         self._processor = QueryContextProcessor(self)
 
     def get_data(
-        self,
-        df: pd.DataFrame,
+        self, df: pd.DataFrame, coltypes: List[str]
```

Comment:
I believe `coltypes` is incorrectly typed here; they're `GenericDataType`, not `str`:
```suggestion
        self, df: pd.DataFrame, coltypes: List[GenericDataType]
```